submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/SubmissionSubmitter.kt

```diff
@@ -43,14 +45,21 @@ class SubmissionSubmitter(
     }
 
     @Suppress("TooGenericExceptionCaught")
-    private fun process(rqt: SubmitRequest): ExtSubmission {
+    private fun processRequest(rqt: SubmitRequest): ExtSubmission {
```

Comment:
I think delete draft call need to be here. I mean we are performing draft operation here already as setting processing status and setting complete status. What make sense to me is have all "draft" business case here and not spread around many clases.